configure.ac

```diff
@@ -56,6 +67,48 @@ AS_CASE([${RKT_STAGE1_USR_FROM}],
 
 AC_SUBST(RKT_STAGE1_USR_FROM)
 
+m4_define(DEFAULT_STAGE1_NAME,coreos.com/rkt/stage1)
```

Comment:
That's an interesting idea because they're actually different things now that we have kvm.

I think it's a pretty big change so it should be discussed in another issue IMO but it's nice this PR centralices the name here.